src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -9,6 +9,8 @@ public static class MetricEventNames
         public const string ApplicationStartLatency = "host.application.start";
         public const string ApplicationInsightsEnabled = "host.applicationinsights.enabled";
         public const string ApplicationInsightsDisabled = "host.applicationinsights.disabled";
+        public const string OpenTelemetryAzMonEnabled = "host.otel.azmon.enabled";
+        public const string OpenTelemetryOtlpEnabled = "host.otel.otlp.enabled";
```

Comment:
HostMetrics also defines some metric event names:

``` 
public const string AppFailureCount = "azure.functions.app_failures";
public const string ActiveInvocationCount = "azure.functions.active_invocations";
public const string StartedInvocationCount = "azure.functions.started_invocations";
```

Raising this for awareness; just want to make sure everything is aligned. @jviau - any concerns on naming? I know we decided on `azure.functions` for the new metrics we added, but do we also need to follow this naming pattern for existing metrics like the ones in this doc?

Reply:
I will follow-up on this.